engine/src/main/java/org/terasology/rendering/assets/skeletalmesh/Bone.java

```diff
@@ -31,7 +30,7 @@
     private String name;
     private int index;
     private Matrix4f relativeTransform = new Matrix4f();
-    private Matrix4f inverseBindMatrix = new Matrix4f(Matrix4f.IDENTITY);
```

Comment:
This seems like those two matrices should be different... why do we joml-migrate both to a plain new matrix?

Reply:
Matricies in JOML are always Identity.

Reply:
yes, but `Matrix4f()` wasn't, was it?

Reply:
it needs to be a zero matrix. umm I don't think there is a way to declare an empty matrix in joml.

Reply:
The initial value is overwritten in the constructor anyways, so this is no problem.